libs/core/web/src/lib/routes/QueryClientRoute.tsx

```diff
@@ -0,0 +1,14 @@
+import { Outlet } from 'react-router-dom';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+const queryClient = new QueryClient({
+  defaultOptions: { queries: { staleTime: 20000 } },
+});
+
+export const QueryClientRoute = () => {
+  return (
+    <QueryClientProvider client={queryClient}>
+      <Outlet />
```

Comment:
why create a QueryClientRoute ? Why not insert the Provider above the app directly ?